packages/ui/components/Module.tsx

```diff
@@ -18,17 +18,22 @@ export const outerModuleVariants = cva(
         default: 'col-span-1',
         lg: 'col-span-1 sm:col-span-2',
       },
+      outline: {
+        true: 'bg-module-outline',
+        false: '',
+      },
     },
     defaultVariants: {
       variant: 'default',
       size: 'default',
+      outline: true,
     },
   }
 );
 
 const innerModuleVariants = cva(
   cn(
-    'rounded-2xl w-full h-full flex align-middle flex-col bg-module',
+    'rounded-[15px] w-full h-full flex align-middle flex-col bg-module',
```

Comment:
happy with a hardcoded pixel value?

Reply:
yeah its the inner radius, has to be hard coded to keep a consistent border width